java/debugger/impl/src/com/intellij/debugger/ui/tree/render/ByteArrayAsStringRenderer.java

```diff
@@ -24,7 +24,7 @@ public String calcLabel(ValueDescriptor descriptor, EvaluationContext evaluation
           Value value = descriptor.getValue();
           if (value instanceof ArrayReference) {
             // TODO: read charset from the target vm
-            byte[] bytes = DebuggerUtilsImpl.readBytesArray(value);
+            byte[] bytes = DebuggerUtilsImpl.fastReadBytesArray(value, evaluationContext.getSuspendContext().getThread());
```

Comment:
does not make much sence here as evaluation is not possible, so you won't be able to invoke String constructor